src/cpp/session/modules/SessionProfiler.R

```diff
@@ -13,15 +13,24 @@
 #
 #
 
+.rs.addFunction("profile_resources", function()
```

Comment:
Minor style thing, but we prefer snake_case for RPC functions, and camelCase for regular `.rs` functions.